commit cad6637e6569825c8c704e6fe1fbc2da4bb91aa6
Author: Francis Ricci <francisjricci@gmail.com>
Date:   Fri Jun 24 20:44:23 2016 +0000

    Don't run TestImageListMultiArchitecture during remote test suite
    
    Reviewers: zturner, clayborg, tfiala
    
    Subscribers: sas, lldb-commits
    
    Differential Revision: http://reviews.llvm.org/D21648
    
    git-svn-id: https://llvm.org/svn/llvm-project/lldb/trunk@273720 91177308-0d34-0410-b5e6-96231b3b80d8

diff --git a/packages/Python/lldbsuite/test/functionalities/object-file/TestImageListMultiArchitecture.py b/packages/Python/lldbsuite/test/functionalities/object-file/TestImageListMultiArchitecture.py
index a62ecb0..0fdc43b 100644
--- a/packages/Python/lldbsuite/test/functionalities/object-file/TestImageListMultiArchitecture.py
+++ b/packages/Python/lldbsuite/test/functionalities/object-file/TestImageListMultiArchitecture.py
@@ -21,6 +21,7 @@ class TestImageListMultiArchitecture(TestBase):
     mydir = TestBase.compute_mydir(__file__)
 
     @no_debug_info_test
+    @skipIfRemote
     def test_image_list_shows_multiple_architectures(self):
         """Test that image list properly shows the correct architecture for a set of different architecture object files."""
         images = {
